packages/analytics/src/analytics.ts

```diff
@@ -0,0 +1,65 @@
+import * as analytics from 'universal-ga';
+import { ILTAnalytics, ILTAnalyticsOptions } from 'lt-analytics';
+
+export class LTAnalytics implements ILTAnalytics {
+  private allTrackersIDs: string[];
+  private options: ILTAnalyticsOptions;
+
+  constructor(options: ILTAnalyticsOptions) {
+    this.options = options;
+    this.allTrackersIDs = this.options.trackers;
+
+    if (!this.allTrackersIDs || this.allTrackersIDs.length < 1) {
+      throw new Error('Configuration Error. You must specify at least one tracker');
+    }
+
+    const defaultTrackingID = this.allTrackersIDs[0];
+    analytics.initialize(defaultTrackingID, options.createOptions);
+
+    this.allTrackersIDs.forEach((tracker) => {
+      analytics.create(tracker, this.getTrackerName(tracker));
+    })
+
+  }
+
+  private getTrackerName(trackerID: string) {
+    const alphaNumericTrackerID = trackerID.split('-').join('');
+    return `${this.options.provider}${alphaNumericTrackerID}`;
+  }
+
+  private shouldTrackEvent(trackerID: string, eventCategory: string) {
+    return this.options.events[trackerID][eventCategory] && this.options.events[trackerID][eventCategory].enabled;
+  }
+
+  private shouldTrackScreenView(trackerID: string, screenName: string) {
+    return this.options.screenViews[trackerID][screenName] && this.options.screenViews[trackerID][screenName].enabled;
+  }
+
+  public set(key: string, value: string) {
```

Comment:
What this function actually sets ? Just to bother you ))) May be we could assign a bit more descriptive name to this function :cat2: